Sources/Starknet/Providers/StarknetProviderProtocol.swift

```diff
@@ -73,6 +73,13 @@ public protocol StarknetProviderProtocol {
     ///  - filter : the conditions used to filter the returned events
     /// - Returns: events matching the conditions in the provided filter and continuation token
     func getEvents(filter: StarknetGetEventsFilter) async throws -> StarknetGetEventsResponse
+
+    /// Get all event objects matching the conditions in the provided filter
+    ///
+    /// - Parameters:
+    ///  - txHash : the hash of the requested transaction
+    /// - Returns: receipt of a transaction identified by given hash
+    func getTransactionReceipt(txHash: Felt) async throws -> StarknetTransactionReceipt
```

Comment:
Cosmetic thing, but I already added `getTransactionBy(hash: Felt)`, so maybe we could use the same naming convention here?

Reply:
Sure, I probably just missed that